app/api/auth/resend-verification/route.ts

```
@@ -0,0 +1,95 @@
+    if (!emailRegex.test(email)) {
+      return NextResponse.json(
+        { error: 'invalid_email', message: 'Invalid email format' },
+        { status: 400 }
+      );
+    }
+
+    // TODO: Check if user exists in database
+    // For now, we'll allow resending for any email
+
+    const existingVerification = pendingVerifications.get(email);
+    if (existingVerification && existingVerification.verified) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)